File: 1.c
#include <stdio.h>
#include <stdlib.h>

// 1. Two Sum

/**
 * Note: The returned array must be malloced, assume caller calls free().
 */
int* twoSum(int* nums, int numsSize, int target) {
    int i, j, flag;
    int *retNums = (int *)malloc(2 * sizeof(int));
    
    flag = 0;
    for(i = 0; i < numsSize; i++) {
        for(j = i + 1; j < numsSize; j++) {
            if(nums[i] + nums[j] == target) {
                retNums[0] = i;
                retNums[1] = j;
                
                flag = 1;
                break;
            }
        }
        
        if(flag == 1) {
            break;
        }
    }
    
    return retNums;
}

int main()
{
    return 0;
}

File: swift/1.swift

// 1. Two Sum

class Solution {
    func twoSum(_ nums: [Int], _ target: Int) -> [Int] {
        var retInts: [Int] = [0, 0]
        let numsLen = nums.count

outLoop: for i in 0..<numsLen {
            for j in i..<numsLen {
                if i != j && (nums[i] + nums[j]) == target {
                    retInts[0] = i;
                    retInts[1] = j;

                    break outLoop;
                }
            }
        }

        return retInts
    }
}